BookingsAPI/Bookings.IntegrationTests/Helper/TestDataManager.cs

```diff
@@ -113,7 +143,7 @@ private CaseType GetCaseTypeFromDb(string caseTypeName)
 
                 if (caseType == null)
```

Comment:
Should this be caseTypeName ??
And string.isnullorempty ???

Reply:
No, caseType is an entity, not a string